pyproj/_crs.pxd

```diff
@@ -9,6 +9,10 @@ cdef extern from "proj_experimental.h":
                                const char* crs_3D_name,
                                const PJ* crs_2D)
 
+cdef extern from "proj_experimental.h":
```

Comment:
The code below can be added to the same block right above.
```suggestion
```

Reply:
Done!